### DISCUSSION TITLE
如何实现用shift键配合鼠标自动取词？

### DISCUSSION THREAD

**Discussion by leiman147** [crimx/ext-saladict#2041] [Q&A] [answered]
当鼠标移动到目标单词时，按shift键自动弹出翻译。这个怎么实现？

**Comment by crimx** [comment, marked as answer]
打开沙[拉查词设置](https://saladict.crimx.com/manual.html#open-setting)，「查词习惯」「普通划词」「鼠标悬浮取词」

<img width="802" alt="image" src="https://github.com/crimx/ext-saladict/assets/6882794/dabb1fea-3fe7-4f91-9b9e-23a9b4a9853a">
